get_notebook.py

```diff
@@ -0,0 +1,8 @@
+import os
+import galaxy
+import shutil
+
+hid = os.environ.get('DATASET_HID', None)
+if hid not in ('None', None):
+    galaxy.get(int(hid))
+    shutil.copy('/import/%s' % hid, '/import/ipython_galaxy_notebook.ipynb')
```

Comment:
Not fully understand what you are referring here. But what we did in Galaxy was to preserve backward compatibility for this release with the intention to cut some old hairs after 15.10 is out.